### PR TITLE
Minor changes in profiling work order output.

### PR DESCRIPTION
**Pull request by hbdeshmukh**
- Now prints query ID along with each work order entry.
- Removed spaces between two columns.

### CODE REVIEW

**Comment by hbdeshmukh** [comment]
@jianqiao Can you please review this PR? It's a minor change.

File: query_execution/Foreman.cpp [hunk 1]
```diff
@@ -238,16 +238,17 @@ void Foreman::printWorkOrderProfilingResults(const std::size_t query_id,
...
     const std::size_t worker_id = std::get<0>(workorder_entry);
     fprintf(out,
-            "%lu, %d, %lu, %lu\n",
+            "%lu,%lu,%d,%lu,%lu\n",
+            query_id,
             worker_id,
             worker_directory_->getNUMANode(worker_id),
-            std::get<1>(workorder_entry),
-            std::get<2>(workorder_entry));
+            std::get<1>(workorder_entry),  // Oeprator ID.
```
Comment (jianqiao) [inline query_execution/Foreman.cpp]: Probably the comment should be `// Operator ID.`

**Comment by jianqiao** [comment]
LGTM except for one minor typo.

**Comment by hbdeshmukh** [comment]
Fixed the typo.

**Pull request merged**